aion/web/elm/src/Room/Notifications.elm

```diff
@@ -17,16 +17,39 @@ addToast toast ( model, cmd ) =
     Toasty.addToast myConfig ToastyMsg toast ( model, cmd )
 
 
+
+-- answer feedback toasts
+
+
 incorrectAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 incorrectAnswerToast =
-    addToast (Toasty.Defaults.Error "Error!" "Wrong Answer")
+    addToast (Toasty.Defaults.Error "Error!" "Wrong answer!")
 
 
 closeAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 closeAnswerToast =
-    addToast (Toasty.Defaults.Warning "Close one!" "Your Answer is Almost Correct")
+    addToast (Toasty.Defaults.Warning "Close one!" "Your answer is almost correct!")
 
 
 correctAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 correctAnswerToast =
-    addToast (Toasty.Defaults.Success "Good Answer!" "Your Answer is Correct")
+    addToast (Toasty.Defaults.Success "Good Answer!" "Your answer is correct!")
+
+
+
+-- question creation
+
+
+questionFormValidationErrorToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
+questionFormValidationErrorToast =
+    addToast (Toasty.Defaults.Error "Error!" "Submission form is not valid!")
+
+
+questionCreationErrorToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
+questionCreationErrorToast =
+    addToast (Toasty.Defaults.Error "Error!" "Failed to create a question.")
+
+
+questionCreationSuccessfulToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
+questionCreationSuccessfulToast =
+    addToast (Toasty.Defaults.Success "Success!" "Question created successfully.")
```

Comment:
I've already did that here: https://github.com/pmrukot/aion/pull/76, will apply necessary changes to this PR and then rebase the one above. My feedback to your suggestion -> Basically it all comes down to taking `addToasts` function higher and storing the message strings in some way. But this level of verbosity that we currently have is not necessarily bad, I think we can live with it for mow.